Corona-Warn-App/src/test/java/de/rki/coronawarnapp/coronatest/qrcode/RapidAntigenQrCodeExtractorTest.kt

```diff
@@ -59,4 +59,9 @@ class RapidAntigenQrCodeExtractorTest : BaseTest() {
     fun `personal data is only valid if complete or completely missing`() {
         shouldThrow<InvalidQRCodeException> { instance.extract(raQrIncompletePersonalData) }
     }
+
+    @Test
+    fun `randon qr code does not crash`() {
```

Comment:
Rename `random qr code does not crash` -> `qr code with empty strings does not crash` ?

Reply:
Removed. I added this test in the validator.